TombEngine/Objects/TR4/Entity/tr4_von_croy.cpp

```diff
@@ -701,8 +702,8 @@ namespace TEN::Entities::TR4
 
 			if (!creature->Flags && enemy != nullptr)
 			{
-				if (item->Animation.FrameNumber > GetAnimData(item).frameBase + 20 &&
-					item->Animation.FrameNumber > GetAnimData(item).frameBase + 45)
+				if (item->Animation.FrameNumber > 20 &&
+					item->Animation.FrameNumber > 45)
```

Comment:
It's a typo, also in previous code to this PR. Should be `item->Animation.FrameNumber < 45`. Proof in tomb4 code: https://github.com/Trxyebeep/tomb4-legacy/blob/a0620f455350aa4947bbcbe18b417b0368e69fad/TOMB4/game/voncroy.cpp#L1835